packages/webapp/src/components/Help/index.jsx

```diff
@@ -11,15 +11,21 @@ import TextArea from '../Form/TextArea';
 import Input from '../Form/Input';
 import Radio from '../Form/Radio';
 import { Label } from '../Typography/index';
+import { useSelector } from 'react-redux';
+import { userFarmSelector } from '../../containers/userFarmSlice';
+import ConfirmModal from '../Modals/Confirm';
 
-export default function PureHelpRequestPage({ onSubmit, goBack, email, phone_number, isLoading }) {
+export default function PureHelpRequestPage({ onSubmit, goBack, isLoading }) {
   const [file, setFile] = useState(null);
+  const [showConfirmModal, setShowConfirmModal] = useState(false);
```

Comment:
I'm not sure what the policy is in this code base for when to use `useState` and when to use `redux`. I saw another form that was doing it this and so I copied the approach because I thought it was simpler